### PR TITLE
Pay 5658

### PR DESCRIPTION
**Pull request by ranjeet-kumarhmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/fees-pay/unprocessed-payment-update/unprocessed-payment-update.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/payment-jobs
+  values:
+    job:
+      image: "hmctspublic.azurecr.io/payment/jobs:prod-7f5ce6e-20220704172137" #{"$imagepolicy": "flux-system:unprocessed-payment-update"}
+      schedule: "* * * * *"
```
Comment (larslunde-hmcts) [inline k8s/namespaces/fees-pay/unprocessed-payment-update/unprocessed-payment-update.yaml:17]: I would change this to a slower schedule, maybe every 30 minutes (*/30 * * * *) as this gets fed through into all the other environments. Keeping the 1minute schedule for testing in demo is fine, but we don't need that in ithc/perftest/aat

Comment (ranjeet-kumarhmcts) [reply k8s/namespaces/fees-pay/unprocessed-payment-update/unprocessed-payment-update.yaml:17]: We will update this schedule once demo testing completes . This schedule is only for testing.

File: apps/fees-pay/automation/kustomization.yaml [hunk 2]
```diff
@@ -67,3 +67,6 @@ resources:
   - ../refund-notifications-job/image-repo.yaml
   - ../refund-notifications-job/image-policy.yaml
   - ../refund-notifications-job/demo-image-policy.yaml
+  - ../unprocessed-payment-update/image-repo.yaml
```
Comment (hannah38) [inline apps/fees-pay/automation/kustomization.yaml:70]: Hey - I think you still want to add the perftest/ithc images here so they're deployed

Comment (ranjeet-kumarhmcts) [reply apps/fees-pay/automation/kustomization.yaml:70]: Added ithc and perftest image

**Review by hannah38 — APPROVED** [review]

**Review by akhil-nuthakki — APPROVED** [review]

**Review by larslunde-hmcts — APPROVED** [review]

**Pull request merged**